.github/workflows/publish.yml

```diff
@@ -78,6 +78,7 @@ jobs:
                   labels: ${{ steps.meta.outputs.labels }}
                   build-args: |
                       "PUBLIC_CONSOLE_MODE=cloud"
+                      "PUBLIC_CONSOLE_FEATURE_FLAGS="sites,csv-import"
```

Comment:
what happens to self-hosted? they won't have access, right?

Reply:
Good point, lets enable both for self-hosted

Reply:
added for all image types except cloud.